mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -377,6 +388,24 @@ private void handleCropResult(@NonNull Intent result) {
         }
     }
 
+    public void snapProfilePicFromCamera() {
+
+        if (Build.VERSION.SDK_INT >= 23 && ContextCompat.checkSelfPermission(this,
+                Manifest.permission.CAMERA) != PackageManager.PERMISSION_GRANTED) {
+
+            // Permission is not granted
+            requestPermissions(new String[]{Manifest.permission.CAMERA}, REQUEST_CAMERA);
```

Comment:
@ReCodee Permission handling is incomplete. Please specify the actions to be taken on the result of the user's request for the permission inside `onRequestPermissionsResult`.

Reply:
It completely slipped from my head, thank you for pointing this out. I've made the changes.